src/api/__tests__/reduxCountriesReducer.test.js

```diff
@@ -0,0 +1,16 @@
+import countriesReducer, { setCounrties } from '../redux/countries/countries';
+
+const initialState = [];
+describe('test redux', () => {
+  it('if is empty array', () => {
+    expect(countriesReducer(initialState, [])).toEqual([]);
+  });
+
+  it('if it is 3 items array', () => {
+    expect(countriesReducer(initialState, setCounrties(['Brazil', 'Germany', 'Afganistan']))).toEqual(['Brazil', 'Germany', 'Afganistan']);
+  });
+
+  it('if return single item', () => {
+    expect(countriesReducer(initialState, setCounrties(['Albania']))).toEqual(['Albania']);
+  });
+});
```

Comment:
- [ ] Some of your tests are failing as seen below. I found duplicates of your tests files which makes it difficult to know the ones that are actually failing. Kindly fix the tests that are failing and also remember that most of them are failing due to path errors. Kindly check your path names because your redux file shows `Redux` but you are using `redux`. Kindly fix the paths

<img width="393" alt="Code_tDm14tEIeh" src="https://user-images.githubusercontent.com/81468675/157462561-076fe3e3-6a62-462d-b98b-49ac44deb14b.png">